src/GlobalUtils.ts

```diff
@@ -47,31 +47,29 @@ function speak(text): void {
 }
 
 /**
- * Import a palette from the given JSON file using dynamic `import()`.
- * The location of the JSON file is provided as a variable.
- * See https://vitejs.dev/guide/features.html#dynamic-import
- *
+ * Load a palette from the given JSON file using `fetch()`. The location of the 
+ * JSON file is provided as a variable. If the loading fails, a console error with
+ * detailed error message is reported.
+ * 
  * @param {String} jsonFilePath - Path of the JSON file to load, without the
  *                                ".json" extension (added herein).
  * @return {JsonPaletteType}    - The palette itself, or `null` if it could not be
  *                                loaded.
  */
-async function importPaletteFromJsonFile (jsonFilePath: string): Promise<JsonPaletteType> {
-  // The "@vite-ignore" comment is used to suppress a warning generated by Vite during
-  // dynamic imports analysis. This warning is reported when both the file path and the
-  // filename are provided in a single variable. Using one single file location variable
-  // enhances the clarity and maintainability of the code, so this comment serves to
-  // mitigate the warning while ensuring the code remains sensible.
-  // Note: the file extension ".json" is required to be concatenated separately for the
-  // dynamic import of JSON files to work properly.
-  // See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations
-  // for supported dynamic import formats.
-  const paletteJson = await import(/* @vite-ignore */`${jsonFilePath}.json`);
-  return paletteJson;
+async function loadPaletteFromJsonFile (jsonFilePath: string): Promise<JsonPaletteType> {
```

Comment:
Good discovery. The commit you referred is within [the pull request #6](https://github.com/inclusive-design/adaptive-palette/pull/6/commits) that you ended up using `import()` rather than `fetch()`. Can you recall why you made the change? I think it might because at the time palette files were part of `/src` so they were bundled by `vite` and `rollup` during the build process. These palette files ended up inaccessible by `fetch()` as `fetch()` only works on publicly accessible files.